src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -359,9 +359,7 @@ void EnBom_Draw(Actor* thisx, PlayState* play) {
     s32 pad;
     EnBom* this = (EnBom*)thisx;
 
-#ifdef OOT_DEBUG
-    if (1) {}
-#endif
+    if (OOT_DEBUG) {}
```

Comment:
I think we should only use the define when we are sure its being used for debug features. We can and do remove if (0)'s all the time through other means, like restructuring how if conditionals are written + early returns. So we should leave the empty ones unless you can find a way to make the debug define usage meaningful

Reply:
actually I missed that in this case the if (1) is ifdeffed out in retail. this one makes sense. but my point still stands for other cases :D

Reply:
Yeah I agree in general. I'm not sure if there's a better way to match this case